war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1265,17 +1329,51 @@ var jenkinsRules = {
         // initial positioning
         Element.observe(window,"load",adjustSticker);
         adjustSticker();
-    }
-};
-/** @deprecated Use {@link Behaviour.specify} instead. */
-var hudsonRules = jenkinsRules; // legacy name
-(function() {
-    var p = 20;
-    for (var selector in jenkinsRules) {
-        Behaviour.specify(selector, 'hudson-behavior', p++, jenkinsRules[selector]);
-        delete jenkinsRules[selector];
-    }
+    });
+
+    /**
+     * Function that provides compatibility to the checkboxes without title on an f:entry
+     *
+     * When a checkbox is generated by setting the title on the f:entry like
+     *     <f:entry field="rebaseBeforePush"title="${%Rebase Before Push}">
+     *         <f:checkbox />
+     *     </f:entry>
+     * This function will copy the title from the .setting-name field to the checkbox label.
+     * It will also move the help button.
+     *
+     * @param {HTMLLabelElement} label
+     */
+    Behaviour.specify('label.js-checkbox-label-empty', 'form-fallbacks', 1000, function(label) {
+        var labelParent = label.parentElement;
+        if (!labelParent.classList.contains('setting-main')) return;
+
+        function findSettingName(formGroup) {
+            for (var i=0; i<formGroup.childNodes.length; i++) {
```

Comment:
Caused [JENKINS-65511](https://issues.jenkins.io/browse/JENKINS-65511) as demonstrated in #6739.